roles/confluent.kafka_connect/tasks/confluent_hub.yml

```diff
@@ -19,9 +19,18 @@
     src: "{{kafka_connect_confluent_hub_client.install_dir}}/bin/confluent-hub"
     dest: "/usr/bin/confluent-hub"
     state: link
-    
+
 - name: "Installing Kafka Connect Connector(s) from Confluent Hub"
   shell: "{{kafka_connect_confluent_hub_client.install_dir}}/bin/confluent-hub install --no-prompt --component-dir {{kafka_connect_confluent_hub_plugins_dest}} {{item}}"
-  with_items: "{{kafka_connect_confluent_hub_plugins}}"
+  loop: "{{kafka_connect_confluent_hub_plugins}}"
+  when: kafka_connect_confluent_hub_plugins|length > 0
+  notify: restart connect distributed
+
+- name: Fix Plugin Folder(s) Permission
```

Comment:
If I follow this correctly, for every plugin folder, you are updating the mode (after doing a little text replacement on the plugin name). Should we also set the group/owner?

Also can you do the mode in hectal (numbers instead of o+rX), everywhere else in our project we use numbers